### DISCUSSION TITLE
Asterisk Speech-to-Text Integration for Deepgram

### DISCUSSION THREAD

**Discussion by vadikat** [deepgram/community#956] [General help] [answered]
Good afternoon, please tell me where to start, I want to integrate deepgram into my asterisk so that it can convert speech into text in real time

**Comment by jpvajda** [comment, marked as answer]
@vadikat  it's best to start with our [Docs for our Streaming API ](https://developers.deepgram.com/docs/getting-started-with-live-streaming-audio)

We don't have any specific Docs on using Deepgram with Asterisk at this time.

You can also check out our different [Starter Apps for Streaming Audio](https://developers.deepgram.com/docs/stt-streaming-starter-apps) 

Finally Our SDKs all contain examples and depending on which language you are developing with you can find those examples in `/examples` directory of our [different SDKs we offer.](https://developers.deepgram.com/docs/deepgram-sdks)
